### PR TITLE
feat: fixed issues in vehicle miles report

### PR DESCRIPTION
**Pull request by saurkarankita**
## Change Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist: (Check only applicable)

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/app/pages/reports/fleet/vehicle-report/activity/activity.component.html [hunk 1]
```diff
@@ -40,8 +40,8 @@
                                     Export<span class="caret"></span></button>
                                 <div class="dropdown-menu" role="menu" x-placement="bottom-start"
                                     style="position: absolute; will-change: transform; top: 0px; left: 0px; transform: translate3d(0px, 38px, 0px);">
-                                    <a class="dropdown-item text-1" (click)="csv()">CSV</a>
-
+                                    <a class="dropdown-item text-1" (click)="fetchFullExport('US') ">US</a>
+                                    <a class="dropdown-item text-1" (click)="fetchFullExport('CAN') ">Canada</a>
```
Comment (FH-sandeepkaur1) [inline src/app/pages/reports/fleet/vehicle-report/activity/activity.component.html:44]: Why 2 types are added here?

File: src/app/pages/reports/fleet/vehicle-report/activity/activity.component.ts [hunk 2]
```diff
@@ -166,63 +127,66 @@ export class ActivityComponent implements OnInit {
...
                 }
+                if (type === 'CAN') {
+                    stateArr = element.canStatesData;
+                } else if (type === 'US') {
+                    stateArr = element.usStatesData;
+                }
+                stateArr.forEach((tripUSState, stateIndex) => {
+                    if (element.provinceData && element.provinceData.length > 0) {
+                        for (let i = 0; i < element.provinceData.length; i++) {
+                            const element2 = element.provinceData[i];
```
Comment (FH-sandeepkaur1) [inline src/app/pages/reports/fleet/vehicle-report/activity/activity.component.ts:157]: Why this code is added here?

**Review by FH-sandeepkaur1 — CHANGES_REQUESTED** [review]

File: src/app/pages/reports/fleet/vehicle-report/activity/activity.component.html (hunk 1 above)
Comment (saurkarankita) [reply src/app/pages/reports/fleet/vehicle-report/activity/activity.component.html:44]: ok i will resolve this . i'm getting province miles reference bcos both report is near by same. not issue i'll change this.

File: src/app/pages/reports/fleet/vehicle-report/activity/activity.component.ts (hunk 2 above)
Comment (saurkarankita) [reply src/app/pages/reports/fleet/vehicle-report/activity/activity.component.ts:157]: ok i will resolve this . i'm getting province miles reference bcos both report is near by same. not issue i'll change this.

**Pull request merged**